doc/source/index.rst

```diff
@@ -23,16 +24,19 @@ SDK
    * `Ansible <https://www.ansible.com>`_
    * `Open Telekom Cloud modules </ansible-collection-cloud>`_
 
-
 CLI
 ---
 
 * `OpenStackClient <https://docs.openstack.org/python-openstackclient/>`_
 * `OpenStackClient Open Telekom Cloud plugins <https://docs.otc-service.com/python-otcextensions>`_
 
-
 Other Drivers and Tools
 -----------------------
 
 * `Docker machine driver <https://github.com/opentelekomcloud/docker-machine-opentelekomcloud>`_
 * `Rancher Open Telekom Cloud CCE driver <https://github.com/opentelekomcloud/kontainer-engine-driver-otc>`_
+
+Cloud Connect Configuration
+---------------------------
+OTC Connect is easy to :doc:`configure <configuration>`. All credentials can be
```

Comment:
I dislike term `OTC Connect`. We can instead write `Open Telekom Cloud tools are easy to configure ...`

Reply:
done